src/vfb_connect/cross_server_tools.py

```diff
@@ -977,39 +984,106 @@ def cypher_query(self, query, return_dataframe=True, verbose=False):
             return pd.DataFrame.from_records(dc)
         return dc
 
-    def get_nt_receptors_in_downstream_neurons(self, upstream_type, downstream_type='neuron', weight=0 , return_dataframe=True, verbose=False):
+    def get_nt_predictions(self, term, verbose=False):
```

Comment:
Do we want this added as a property of the neuron or is the existing N.capable_of and X.related_terms enough?

Reply:
Might be good to have a more specific, explicitly-named property

Reply:
neurotransmitter and neurotransmitter_prediction? (based on the Capable_of having a confidence score)?

Reply:
we can filter by the same list just don't like to add more query time to term loading so easier if we can use the data we have

Reply:
maybe do neurotransmitter_predictions for individuals and classes (will need to be a list if using at the class level) and only get known_neurotransmitters for classes (also needs to be a list)

Reply:
nb predictions will be the GO terms and known will be the neo labels, unless you want to do some kind of mapping.

Reply:
I'll get this merged for now anyway